.github/workflows/task-test.yml

```diff
@@ -172,11 +201,7 @@ jobs:
         if: >
```

Comment:
consider updating the comment here

Reply:
You mean to remove the failure condition?